cmd/platform/export.go

```diff
@@ -232,11 +233,18 @@ func initApiClient(ctx context.Context) (*sdk.Client, error) {
 		return nil, fmt.Errorf("unrecognized PingOne Region: %q. Must be one of: %q, %q, %q, %q", region, connector.ENUMREGION_AP, connector.ENUMREGION_CA, connector.ENUMREGION_EU, connector.ENUMREGION_NA)
 	}
 
+	userAgent := fmt.Sprintf("pingctl/%s", version)
+
+	if v := strings.TrimSpace(os.Getenv("PINGCTL_PINGONE_APPEND_USER_AGENT")); v != "" {
```

Comment:
Is this environment variable meant to be configurable via all three methods? (config file, env var, or parameter?)

Reply:
The intention is that we don't include this env var in viper parameter logic as it's an edge case for integration in CICD pipelines.  The vast majority of users will not need to use this variable, but there is an internal use case that does

Reply:
Sounds good 👍